add-on/panels/js/roomToc.jsx

```diff
@@ -0,0 +1,61 @@
+var loop = loop || {};
+
+loop.toc = (function() {
+  "use strict";
+  var tocViews = loop.shared.toc;
+  var sharedActions = loop.shared.actions;
+  var sharedUtils = loop.shared.utils;
+
+  function init() {
+    loop.shared.utils.getBoolPreference = function foo() {};
+
+      var dispatcher = new loop.Dispatcher();
+      var sdkDriver = new loop.OTSdkDriver({
```

Comment:
I've not tested this, so I'm assuming the conversation window items are here because of current non-separation of ToC & Sidebar. It feels like we probably want a bug to confirm separation of the two.